src/main/java/org/springframework/social/partnercenter/api/billing/pricing/OfferTerm.java

```diff
@@ -14,7 +14,7 @@
 	@JsonProperty("discount")
 	private Double discount;
 	private List<String> excludedMeterIds;
-	@JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss", timezone = "UTC")
+	@JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss['Z']", timezone = "UTC")
```

Comment:
Can a new test be added to show this change is effective?

Reply:
> Can a new test be added to cover this new format?

https://github.com/SpringSocialMicrosoft/spring-social-partnercenter/blob/b94673116ddc3bbb0c96ff67fc56ca4abaeec1e7/src/test/java/org/springframework/social/partnercenter/api/billing/pricing/PricingOperationsTest.java#L35

in this line, it's covered already